eng/Directory.Build.Data.props

```diff
@@ -39,9 +39,10 @@
     <IsMgmtClientLibrary Condition="'$(IsMgmtClientLibrary)' == '' and ($(MSBuildProjectName.StartsWith('Azure.Management.')) or $(MSBuildProjectName.StartsWith('Azure.ResourceManager.')))">true</IsMgmtClientLibrary>
     <IsTestProject Condition="$(MSBuildProjectName.EndsWith('.Tests'))">true</IsTestProject>
     <IsSamplesProject Condition="$(MSBuildProjectName.EndsWith('.Samples'))">true</IsSamplesProject>
+    <IsStressProject Condition="$(MSBuildProjectName.EndsWith('.Stress'))">true</IsStressProject>
```

Comment:
I'm good with this convention but it looks like it will be set for both the the stress framework library as well as the stress tests projects. There might be some cases where you need to distinguish between the 2.

Reply:
This was intentional since their settings should be the same, I can add `IsStressSupportProject` later if needed.  I'd prefer to keep perf and stress separate from "test" since their settings are different.